3rdParty/README_maintainers.md

```diff
@@ -126,72 +126,15 @@ https://github.com/lz4/lz4
 
 ## rocksdb
 
-(6.29.0, upstream commit e8f116deabc601a5eae011f2fd8a38d02d87be8d)
+v6.29.5
```

Comment:
Isn't this PR upgrading the RocksDB version to 7.2?

Reply:
Fixed!